templates/netlify/remix.init/remix.env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="@remix-run/dev" />
```

Comment:
If I'm correct, this file isn't necessary with the `deno` runtime.
So instead of overriding it in `remix.init`, we should remove it when people choose for Netlify Edge

CC/ @pcattori

Reply:
That's right. VS Code reports 0 type warnings/errors in for me in `templates/deno` when using the Deno extension. So this file is not necessary for that context. Again, this assumes users have the Deno extension installed and enabled.

Reply:
We use it if they choose Netlify Functions instead of Netlify Edge. I can remove it via remix.init for Netlify Edge.